app/Http/Requests/AnswerFormRequest.php

```diff
@@ -246,6 +246,14 @@ protected function prepareForValidation()
             }
         });
 
+        $countryCodeMapper = json_decode(file_get_contents(resource_path('data/country_code_mapper.json')), true);
```

Comment:
We already did `collect(properties)` to improve speed we should avoid doing this many times. You can run the `->each` right after the `->filter()`